src/libfuncs/pedersen.rs

```diff
@@ -62,7 +63,7 @@ pub fn build_pedersen<'ctx>(
     let pedersen_builtin =
         super::increment_builtin_counter(context, entry, location, entry.argument(0)?.into())?;
 
-    let felt252_ty = registry.build_type(
+    let value_ty = registry.build_type(
```

Comment:
I'm curious about this name change. Doesn't this name convey less information about the variable?

Reply:
Hi @azteca1998  sorry for the late answer. Actually i changed that because the ''base'' PR that defines the BlockExtTrait modified used at some point  `value_ty` instead of `u128_ty`. But yeah, it feels better with `felt252_ty` here, will change it back.